pkg/kubernetes/app.js

```diff
@@ -118,6 +152,16 @@ define([
             };
         })
 
+        .filter("namespaceDisplay", function() {
```

Comment:
I'll remove it, but that was why i put it in, if you list projects 'oc get projects' it does show the 'pretty name' listing namepsaces doesn't. 

Not that we care but web console also only shows pretty name.